test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionProgressIT.java

```diff
@@ -124,7 +130,14 @@ public Thread startChecker() {
    * Check running compaction progress.
    */
   private void checkRunning() throws TException {
-    var ecList = getRunningCompactions(getCluster().getServerContext());
+
+    ServerContext ctx = getCluster().getServerContext();
+    Optional<HostAndPort> coordinatorHost = ExternalCompactionUtil.findCompactionCoordinator(ctx);
```

Comment:
I removed the call to `ExternalCompactionUtil.findCompactionCoordinator` from `getRunningCompactions`and `getCompletedCompactions` so that we were not hitting ZK so much. In some of the tests these methods are called in a loop. This change reduces the calls to ZK.